inference-engine/src/readers/ir_reader/ie_ir_parser.cpp

```diff
@@ -704,8 +704,10 @@ std::shared_ptr<ngraph::Node> V10Parser::XmlDeserializer::createNode(
         ngraphNode->set_friendly_name(params.name);
         ngraphNode->set_arguments(inputs);
         XmlDeserializer visitor(node, weights, opsets);
-        if (ngraphNode->visit_attributes(visitor))
+        if (ngraphNode->visit_attributes(visitor)) {
             ngraphNode->constructor_validate_and_infer_types();
+        }
+        ngraphNode = ngraphNode->clone_with_new_inputs(ngraphNode->input_values());
```

Comment:
@ilyachur , @GlebKazantaev , does `clone` should be always call or only inside above `if`?

Reply:
It must be called only in case when visit_attributes is applied. So the place where you put this clone is correct.

Reply:
> inside above `if`

I mean this:
```
if (ngraphNode->visit_attributes(visitor)) {
    ngraphNode->constructor_validate_and_infer_types();	                
    ngraphNode = ngraphNode->clone_with_new_inputs(ngraphNode->input_values());
}
```

Reply:
> It must be called only in case when visit_attributes is applied. So the place where you put this clone is correct.

  @GlebKazantaev  `visit_attributes` is applied inside `if (ngraphNode->visit_attributes(visitor)) {`. Looks like we should move `clone_with_new_inputs` there, is it right?

Reply:
The initial problem is that we create node with default c-tor and then call `set_arguments`. So we need to call `clone_with_new_inputs` always after node creation even if `visit_attributes` returns false.

Reply:
got it.